activiti-cloud-services-audit-jpa/src/main/java/org/activiti/cloud/services/audit/jpa/controllers/AuditEventsCleanUpController.java

```diff
@@ -0,0 +1,65 @@
+package org.activiti.cloud.services.audit.jpa.controllers;
+
+import org.activiti.cloud.api.model.shared.events.CloudRuntimeEvent;
+import org.activiti.cloud.services.audit.api.assembler.EventResourceAssembler;
+import org.activiti.cloud.services.audit.api.converters.APIEventToEntityConverters;
+import org.activiti.cloud.services.audit.api.resources.EventsRelProvider;
+import org.activiti.cloud.services.audit.jpa.events.AuditEventEntity;
+import org.activiti.cloud.services.audit.jpa.repository.EventsRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.hateoas.MediaTypes;
+import org.springframework.hateoas.Resource;
+import org.springframework.hateoas.Resources;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+@ConditionalOnProperty(name = "activiti.rest.enable-clean-up", havingValue = "true")
+@RestController
+@RequestMapping(
+        value = "/admin/clean-up/v1/" + EventsRelProvider.COLLECTION_RESOURCE_REL,
```

Comment:
@miguelruizdev @erdemedeiros I don't like this path.. it is messing things up.. 

- /admin/v1/ -> that is fixed ..we cannot add things in the middle...
- clean-up can be cleanup, but the term is not 100% correct. I think that we need something like /export?delete=true will be more accurate..

@miguelruizdev @erdemedeiros Opinions?

Reply:
and here the path should be /admin/v1/events/export ? or /admin/v1/export if there is no other resource in audit.

Reply:
I think that this was discussed once in the main thread in the issue: https://github.com/Activiti/Activiti/issues/2173#issuecomment-467880422
It causes problems with the **HTTP** verb we would use for the endpoint (since an export would be basically a **GET** but with the flag delete=true would behave as a **DELETE**)

Reply:
Regarding the path, `/admin/v1/` makes perfect sense to be fixed, so whatever the name we agree on is (`export` / `delete`) , it should be at the end

Reply:
yes.. but also consider the resource.. so if there is a resource associated it needs to be inside the resource.

Reply:
@salaboy, as @miguelruizdev mentioned it sounds confusing to me the flag `delete=true`, as the behaviour becomes inconsistent with HTTP verb. In addition, currently someone can already return all the events using get endpoint. Initially I was thinking to use simply `DELETE /admin/v1/events`: same path as GET, but with DELETE verb, but I think people wanted it with a different path to avoid "calling it by mistake", that's why `cleanup` came in. @salaboy we can have quick catch up on this if you want.